iterable-extension/src/main/java/com/mparticle/ext/iterable/IterableExtension.java

```diff
@@ -434,9 +531,6 @@ public void processCustomEvent(CustomEvent event) throws IOException {
                 }
             }
         }
-        //TODO use custom flags to set campaign and template id
```

Comment:
Is this still a todo item?

Reply:
I think we can revisit this in the future if there's a customer request for it. Pretty ugly to have it as a TODO in the code so I removed it.